### ISSUE TITLE
Lack of error and confirmation messages [Storage Limit]

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1790]
Inside the admin panel (User storage), when trying to set a storage limit that doesn't contain a valid number, I don't get any type of error. The expected behavior is that I should have a descriptive error message( eg. Please enter a valid number)
Also, the confirmation message when updating the storage limit is missing.
